src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 5;

```

Comment:
```suggestion
    private static final int FIGURE_COUNT = 5;
```
якщо змінну не потрібно використовувати поза межами класу хорошою практикою вважається використовувати якумога жорсткіший модифікатор доступу